tests/integration/steps_start_chains.go

```diff
@@ -123,30 +108,12 @@ func stepsStartChains(consumerName string, setupTransferChan bool) []Step {
 				},
 			},
 		},
-		{
```

Comment:
Is there a reason this was removed? Seems like a useful check

Reply:
It made tracking validator balances more difficult. I later steps. I think I had some issues with democracy and shutting down a consumer